test/collaboration.test.js

```diff
@@ -452,32 +445,36 @@ describe('slate operations propagate between editors', () => {
 
       // Verify initial states.
       const inputAsJSON = input.map(nodeToJSON);
-      expect(getSlateDocAsJSON(src)).toStrictEqual(inputAsJSON);
-      expect(getSyncDocAsJSON(src)).toStrictEqual(inputAsJSON);
-      expect(getSyncDocAsJSON(dst)).toStrictEqual(inputAsJSON);
-      expect(getSlateDocAsJSON(dst)).toStrictEqual(inputAsJSON);
-
-      // Allow for multiple rounds of applying transforms and verifying state.
-      while (rest.length > 0) {
-        let transforms, output;
-        [transforms, output, ...rest] = rest;
+      const inputValue = Value.create({
+        document: { nodes: inputAsJSON },
+      });
+      expect(src.slateDoc.document.nodes.toArray().map(nodeToJSON)).toStrictEqual(inputAsJSON);
+      expect(toSlateDoc(src.syncDoc).toJSON()).toEqual(inputValue.toJSON());
+      expect(toSlateDoc(dst.syncDoc).toJSON()).toEqual(inputValue.toJSON());
```

Comment:
Are you intentionally dropping the `getSlateDocAsJSON` and `getSyncDocAsJSON` helpers? I had added them because I thought they helped improve the readability of these tests ...

Reply:
Good point, will add it back.